src/styles/Card.js

```diff
@@ -110,6 +111,24 @@ const ContextStyle = styled(Card)`
   a {
     color: ${colors.lighterGrey};
   }
+
+  .languages {
+    margin-top: 20px;
+    
+    .dot {
+      font-size: 25px;
+      margin: 0;
+      vertical-align: sub;
+    }
+    
+    .name {
+      color: black;
+    }
+    
+    .more {
+      color: grey;
```

Comment:
Define color as a variable in "./variables.js" file and include here, eg. - 
${colors.grey}